en/docs/insights/choreo-insights-api.md

```diff
@@ -0,0 +1,4214 @@
+<td valign="top"><a href="#string">String</a></td>
+<td>
+
+Application Id.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>applicationName</strong></td>
+<td valign="top"><a href="#string">String</a></td>
+<td>
+
+Application name.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>applicationOwner</strong></td>
+<td valign="top"><a href="#string">String</a></td>
+<td>
+
+Application owner name.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>reason</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Error reason.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>count</strong></td>
+<td valign="top"><a href="#int">Int</a>!</td>
+<td>
+
+Error count.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### ErrorStatusCodeCategoryCounts
+
+Represents errors by main status code category(4xx or 5xx) and the total error count.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>_4xx</strong></td>
+<td valign="top"><a href="#int">Int</a></td>
+<td>
+
+Number of client errors(4xx response codes).
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>_5xx</strong></td>
+<td valign="top"><a href="#int">Int</a></td>
+<td>
+
+Number of server errors(5xx response codes).
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>total</strong></td>
+<td valign="top"><a href="#int">Int</a></td>
+<td>
+
+Total of client and server errors.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### ErrorSummary
+
+Provides unsuccessful(4xx and 5xx response code) usage summary across all the APIs.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>summary</strong></td>
+<td valign="top">[<a href="#errorvalue">ErrorValue</a>]!</td>
+<td>
+
+List containing unsuccessful usage values. Returns an empty array if no data available.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>granularity</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Granularity value used for data retrieval.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### ErrorValue
+
+Represents unsuccessful request count(4xx and 5xx response code) for a single timestamp. Timestamp is calculated
+according to the provided granularity value.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>timeSpan</strong></td>
+<td valign="top"><a href="#string">String</a></td>
+<td>
+
+Represents a timestamp value calculated according to the used granularity value. For an example if the used
```

Comment:
When you say "used granularity value" it is not clear as to where it is used and by what. Please check whether the following suggestion gives the same meaning.

```suggestion
Represents a timestamp value that is calculated based on the specified granularity value. For example, if the granularity value is `1d`, the timestamp represents the start of the day calculated adhering to the timezone provided in the time filter.
e.g., timeSpan: '2021-06-21T00:00:00.0000000+05:30'
```